settings.gradle

```diff
@@ -4,6 +4,7 @@ include ":opentelemetry-all"
 include ":opentelemetry-api"
 include ":opentelemetry-contrib-runtime-metrics"
 include ":opentelemetry-contrib-trace-utils"
+include ":opentelemetry-examples"
```

Comment:
I am confused about the gradle config, for jaeger example you used a gradle config for that project, but here you added to the main config. I would suggest to have one gradle config per the whole examples directory that depends on the released otel artifacts, similar to https://github.com/census-instrumentation/opencensus-java/tree/master/examples

Reply:
The idea to include the examples as part of the main project originate from the discussion in https://github.com/open-telemetry/opentelemetry-java/issues/692#issuecomment-565146533.
In this way, any change to the API that breaks the compatibility will fail the build and it requires to update the examples.

Reply:
I do think it is better to keep the examples separated: shows a clear example on how to configure a project that depends on the otel library.

I do understand the concern about examples being outdated but I think in OpenCensus for example we did a good job keeping them up to date.

Reply:
> shows a clear example on how to configure a project that depends on the otel library.

Actually this is a fair point. +1

Reply:
I agree. In 5ee081b, I removed the example from the main build and restored it as standalone project.